{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-11379", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": [{"vendor_name": "radare", "product": {"product_data": [{"product_name": "radare2", "version": {"version_data": [{"version_value": "2.5.0", "version_affected": "="}]}}]}}]}}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-125"}]}]}, "references": {"reference_data": [{"url": "https://github.com/radare/radare2/commit/4e1cf0d3e6f6fe2552a269def0af1cd2403e266c", "name": "https://github.com/radare/radare2/commit/4e1cf0d3e6f6fe2552a269def0af1cd2403e266c", "refsource": "MISC", "tags": ["Patch", "Third Party Advisory"]}, {"url": "https://github.com/radare/radare2/issues/9926", "name": "https://github.com/radare/radare2/issues/9926", "refsource": "MISC", "tags": ["Third Party Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": "The get_debug_info() function in radare2 2.5.0 allows remote attackers to cause a denial of service (heap-based out-of-bounds read and application crash) via a crafted PE file."}]}}
